script/src/verify.rs

```diff
@@ -226,39 +227,31 @@ impl<'a, DL: DataLoader> TransactionScriptsVerifier<'a, DL> {
     }
 
     // Extracts actual script binary either in dep cells.
-    fn extract_script(&self, script: &'a Script) -> Result<Bytes, ScriptError> {
+    fn extract_script(&self, script: &'a Script) -> Result<Bytes, Error> {
         match script.hash_type {
             ScriptHashType::Data => {
                 if let Some(data) = self.binaries_by_data_hash.get(&script.code_hash) {
                     Ok(data.to_owned())
                 } else {
-                    Err(ScriptError::InvalidCodeHash)
+                    Err(ScriptError::InvalidCodeHash)?
                 }
             }
             ScriptHashType::Type => {
                 if let Some((data, multiple)) = self.binaries_by_type_hash.get(&script.code_hash) {
                     if *multiple {
-                        Err(ScriptError::MultipleMatches)
+                        Err(ScriptError::MultipleMatches)?
                     } else {
                         Ok(data.to_owned())
                     }
                 } else {
-                    Err(ScriptError::InvalidCodeHash)
+                    Err(ScriptError::InvalidCodeHash)?
                 }
             }
         }
     }
 
-    pub fn verify(&self, max_cycles: Cycle) -> Result<Cycle, ScriptError> {
+    pub fn verify(&self, max_cycles: Cycle) -> Result<Cycle, Error> {
         let mut cycles: Cycle = 0;
-        // Check if all inputs are resolved correctly
-        if self
-            .resolved_inputs()
-            .iter()
-            .any(|input| input.cell.is_none())
-        {
-            return Err(ScriptError::NoScript);
-        }
 
```

Comment:
Check if all inputs are resolved correctly

Reply:
This part will be removed in PR https://github.com/nervosnetwork/ckb/pull/1331/ . So I remove it here. I think it is OK.

https://github.com/nervosnetwork/ckb/pull/1331/files#diff-99bdd708d07843aa77a343e5e906e76fL254